.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+image:
+  file: .devcontainer/.gitpod.Dockerfile
+  context: .devcontainer
+
+tasks:
+  - init: >
+      git submodule init
+      && git submodule update
+      && ./autogen.sh
+      && ./configure --enable-ccache --enable-tracy --enable-tracy-capture --enable-tracy-gui
+      && bear make
+      && make check
+  - command: echo "Welcome to the stellar-core Gitpod development environment."
+
+vscode:
+  extensions:
+    - llvm-vs-code-extensions.vscode-clangd@0.1.5:VVxg4kW5VbJUP3ysCuG3Fg==
```

Comment:
> Do we need to install this extension by default?

I wanted code navigation, hover text, live compiler errors, and formatting according to our standards to work out of the box -- I was targeting the closest thing I could get to a single click and starting to develop.  People could, of course, install it if we didn't.  It'd just be something that they'd have to discover instead of something that was already there for them.



> Or at least not specify any version string (I don't like having hardcoded version strings)

The version string form is the one that Gitpod generates when you install the extension via the GUI, and I just left it.  I'm not sure if there's any reason that it's necessary or advantageous; I would prefer the un-versioned version myself if it works.